### DISCUSSION TITLE
MPO Breakout to multiple devices?

### DISCUSSION THREAD

**Discussion by tomrockwell** [netbox-community/netbox#12369] [Q&A] [answered]
Hi,

On Netbox v3.4.8. Trying to model fiber cabling, specifically MPO to 4x LC pair breakout harness.

The connection is from a QSFP switch port to 4 NIC ports on different servers. In the web interface, can make a connection to multiple interfaces on one device, but I don't see how to get breakout to multiple devices :-(

Is this supposed to work?

I'd be happy to enter these connections via bulk upload if it is just a web interface limitation.

Thanks!

**Comment by candlerb** [comment, marked as answer]
You should be able to do it in the web interface: select from the 'device' dropdown, add an interface, change the 'device' dropdown, add an interface from that device, and so on.

The displayed set of connected interfaces is confusing, because it only shows the local name of each interface, not the (device name + interface name) pair: for example, if a cable terminates on device1 eth0 and device2 eth0, it will just show "(eth0) (eth0)".

For more background, see #10841.

**Comment by tomrockwell** [comment]
Thanks for the help. Isn't always easy to find existing discussion. I see the comment that a UI update to address this is targeted for 3.5.
